gap/Magic.gd

```diff
@@ -323,11 +323,17 @@
 #!
 #!         </List>
 #!     </Item>
-## This is the maketest part. Still under construction.
+#!     <Mark><A>example_test</A></Mark>
+#!     <Item>
+#!         If set to <K>true</K>, then all manual examples are extracted and placed
+#!         into files <F>tst/PACKAGENAME01.tst</F>,  <F>tst/PACKAGENAME02.tst</F>, ...
+#!         and so on, with one file for each chapter.
+#!     </Item>
+## TODO: should we mark maketest as deprecated and recommend extract_examples instead?
```

Comment:
I think we should, yes.
Add the beginning, we could write something like:
> This option is deprecated. To be more compatible with the GAP test framework, please use `example_test`.

Reply:
I've added some text in this vein. Feel free to merge.